`npm install ` throw Error, while run 'prepublish' , caused by `check-es3-syntax`

> node v7.0, npm V3.10.8 (the newest)
- first, i clone this repo
- second, cd into this folder, run `npm install`
- got a error and install deps faild, here is the detail

``` js
There are differences between input version and the ES3 version
/Users/jiangmu/github/redux/dist/redux.min.js
npm info lifecycle redux@3.6.0~prepublish: Failed to exec prepublish script
npm ERR! Darwin 16.1.0
npm ERR! argv "/Users/jiangmu/.nvm/versions/node/v7.0.0/bin/node" "/Users/jiangmu/.nvm/versions/node/v7.0.0/bin/npm" "i" "-dddd"
npm ERR! node v7.0.0
npm ERR! npm  v3.10.8
npm ERR! code ELIFECYCLE
npm ERR! redux@3.6.0 prepublish: `npm run clean && npm run check:src && npm run build && check-es3-syntax lib/ dist/ --kill --print`
npm ERR! Exit status 255
npm ERR!
npm ERR! Failed at the redux@3.6.0 prepublish script 'npm run clean && npm run check:src && npm run build && check-es3-syntax lib/ dist/ --kill --print'.
```

--- a/webpack.config.js
+++ b/webpack.config.js
@@ -28,7 +28,14 @@ if (env === 'production') {
         pure_getters: true,
         unsafe: true,
         unsafe_comps: true,
-        warnings: false
+        warnings: false,
+        screw_ie8: false
+      },
+      mangle: {
+        screw_ie8: false
+      },
+      output: {
+        screw_ie8: false
       }
     })
   )

